server/zoom/meeting.go

```diff
@@ -4,7 +4,7 @@
 package zoom
 
 // Meeting is defined at https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meeting
-type Meeting struct {
+type Meeting struct { // nolint: govet
```

Comment:
What linter error does this ignore?

Reply:
a bit hard to organize the struct, if you want to try I appreciate